README.md

```diff
@@ -1 +1,55 @@
-# ICT4DA
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# ICT4DA : Information and Communication Development for Africa Research Conference website
+
+# Project Name
+
+## Description the project
+This project is a call for papers research conference website. It is a capstone project from microverse from Module#1. The capstone project is supposed to be developed using html, css, javascript, soft skills, git, github, github flow, and etc gained from the past four weeks tight learning. The project developed is supposed to stick to a website card given from graphic designers though the title of the conference and the contents of the website differ. The project developed is responsive designed and developed with the mobile first approach. First the mobile version is developed. The mobile version is developed for mobile devices of which their screen sizes are less than 768px. While developing the mobile version, the pop menus are also designed and developed. The mobile menus are not displayed  at the begining. One can see the menu items by clicking the 'hamburger' icon at the top-left corner of the website. The front-end of the project is developed using html, css, javascript scripts.
+  The wbsite is dynamic as some html elements are manipulated by the javascript code using DOM api.
+
+
+
+## Built With
+
+- HTML
+- CSS
+- Javascript
+- Technologies used: VS code editor, git, github, github flow
+
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** I think it would be better if you add the `## Getting started` section in your **README** including some information on how to get the copy of the repo locally and even access it. I suggest a look at **[this template](https://gist.github.com/tresorsawasawa/d10e8905a56c2bb3ec4c81ac58684d0c)** to help you how to fill the `## Getting started` section. I know you can do that :muscle: :muscle: